plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/client/PinotClient.java

```diff
@@ -114,11 +116,12 @@
     private static final String TABLE_SCHEMA_API_TEMPLATE = "tables/%s/schema";
     private static final String ROUTING_TABLE_API_TEMPLATE = "debug/routingTable/%s";
     private static final String TIME_BOUNDARY_API_TEMPLATE = "debug/timeBoundary/%s";
-    private static final String QUERY_URL_TEMPLATE = "http://%s/query/sql";
+    private static final String QUERY_URL_TEMPLATE = "query/sql";
```

Comment:
This is no longer a "template". Rename the constant to, for instance, `QUERY_URL_PATH`